content/docs/visualization/grafana.md

```diff
@@ -25,15 +25,23 @@ By default, Grafana will be listening on
 
 ### Creating a Prometheus data source
 
-To create a Prometheus data source:
+First, we need to get Prometheus running and listening on port 9090. The easiest way to do this is via Docker:
```

Comment:
These are docs to explain Grafana configuration, not docs on running Prometheus or using Docker. You can presume that Prometheus is already running.

Reply:
ok I have removed that section now